src/writer/abstract.py

```diff
@@ -0,0 +1,16 @@
+"""
+
+Abstract templates.
+
+They're used to register new component templates, based on existing frontend templates and a backend-provided component definition. 
+
+"""
+
+from typing import Dict
+
+from writer.ss_types import AbstractTemplate
+
+templates:Dict[str, AbstractTemplate]  = {}
```

Comment:
I would advocate to use a list here (see my other message about the design of templateMap).